rest-model/src/main/java/com/gentics/mesh/core/rest/plugin/PluginResponse.java

```diff
@@ -22,6 +22,10 @@
 	@JsonPropertyDescription("Manifest of the plugin")
 	private PluginManifest manifest;
 
+	@JsonProperty(required = true)
+	@JsonPropertyDescription("Status of the plugin")
+	private PluginStatus status;
+
 	public PluginResponse() {
```

Comment:
Maybe we should include the potential error message on failures (maybe a security concern although admin perm is needed)
Otherwise we could include the last successful or the error phase in which the plugin failed.

@mephinet